src/EnergyPlus/HeatBalFiniteDiffManager.hh

```diff
@@ -163,9 +164,7 @@ namespace HeatBalFiniteDiffManager {
         Array1D<Real64> heatSourceEMSFluxEnergyLayerReport;
 
         // Default Constructor
-        SurfaceDataFD()
-            : SourceNodeNum(0), QSource(0.0), GSloopCounter(0), GSloopErrorCount(0), MaxNodeDelTemp(0.0), EnthalpyM(0.0), EnthalpyF(0.0),
-              PhaseChangeState(0)
+        SurfaceDataFD() : SourceNodeNum(0), QSource(0.0), GSloopCounter(0), MaxNodeDelTemp(0.0), EnthalpyM(0.0), EnthalpyF(0.0), PhaseChangeState(0)
```

Comment:
In the future, just go ahead and move these initial values from this default constructor up to where the variables are declared, and delete this empty constructor.  No biggie now.